pkg/pillar/types/zedroutertypes.go

```diff
@@ -2889,3 +2913,59 @@ func (status OnboardingStatus) LogDelete(logBase *base.LogObject) {
 func (status OnboardingStatus) LogKey() string {
 	return string(base.OnboardingStatusLogType) + "-" + status.Key()
 }
+
+// NetworkInstanceIPAddrMapInfo :
+type NetworkInstanceIPAddrMapInfo struct {
+	// Collection of address assignments; from MAC address to IP address
+	// and IP Address to MAC
+	IPAssignments  map[string]net.IP
+	MacAssignments map[string]string
+}
+
+// NetworkInstanceAppNumMapInfo :
+type NetworkInstanceAppNumMapInfo struct {
```

Comment:
Why do we need this struct? The persistent NetworkInstanceAppNumMap should be sufficient for the logic to work. Anything else might be an optimization which I don't think we need now.